### ISSUE TITLE
security.googleblog.com: breakage

### ISSUE THREAD

**Issue by joshtriplett** [uBlockOrigin/uAssets#25209]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://security.googleblog.com/2024/09/deploying-rust-in-existing-firmware.html`

### Category

breakage

### Description

With uBlock Origin enabled, https://security.googleblog.com/2024/09/deploying-rust-in-existing-firmware.html shows no article content.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

With uBlock Origin enabled:
![image](https://github.com/user-attachments/assets/0402f9c5-8bdd-46d7-b062-6d7d5feffc03)

With uBlock Origin disabled:
![image](https://github.com/user-attachments/assets/3411de05-a50f-4c40-882d-934995c517e2)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Firefox: 129
filterset (summary):
 network: 124881
 cosmetic: 48063
 scriptlet: 20739
 html: 1952
listset (total-discarded, last-updated):
 removed:
  urlhaus-1: null
 added:
  plowe-0: 3547-994, now
  ublock-quick-fixes: 204-79, now
 default:
  user-filters: 0-0, never
  ublock-filters: 40053-138, now
  ublock-badware: 10981-6, now
  ublock-privacy: 1116-4, now
  ublock-unbreak: 2530-1, now
  easylist: 85798-184, now
  easyprivacy: 53026-63, now
filterset (user): [empty]
trustedset:
 added: [array of 11 redacted]
userSettings:
 prefetchingDisabled: false
hiddenSettings: [none]
supportStats:
 allReadyAfter: 14150 ms (selfie)
 maxAssetCacheWait: 71 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 2
 network:
  doubleclick.net: 1
  googletagmanager.com: 1
 extended:
  ##+js(trusted-replace-node-text, script, (function serverContrac…
  ##+js(adjust-setTimeout, [native code], 17000, 0.001)
  ##+js(json-prune-fetch-response, playerAds adPlacements adSlots …
  ##+js(json-prune-xhr-response, playerAds adPlacements adSlots pl…
  ##+js(set-constant, ytInitialPlayerResponse.playerAds, undefined…
  ##+js(set-constant, ytInitialPlayerResponse.adPlacements, undefi…
  ##+js(set-constant, ytInitialPlayerResponse.adSlots, undefined)
  ##+js(set-constant, playerResponse.adPlacements, undefined)
  ##+js(json-prune-fetch-response, reelWatchSequenceResponse.entri…
```
</details>

**Comment by stephenhawk8054** [comment]
Confirm the page is broken for me too without using any extensions

![image](https://github.com/user-attachments/assets/f1669b6e-7f34-4367-9a26-7d95bf78d238)

**Comment by joshtriplett** [comment]
Sigh. Before reporting this, I tested this multiple times, and each time it consistently worked with uBlock Origin disabled and didn't work with uBlock Origin enabled. However, it *does* look like it's simply inconsistent whether it loads or not, and by loading it several more times I've now observed breakage with uBlock Origin disabled and functioning with uBlock Origin enabled. Apparently I just had a series of *several* successes/failures that were each correlated with the uBlock Origin status.

Closing this, and sorry for what ended up being a red herring.
